### ISSUE TITLE
Simple UML class diagram 

### ISSUE THREAD

**Issue by PeterParslow** [agiorguk/gemini#170]
See https://github.com/agiorguk/gemini/issues/24#issuecomment-811914444
"I think the informal pictorial UML class relationship might help address this. The class diagram should be added to the document" @6footdestiny : I was thinking today that the simple UML overview of GEMINI would help
I think you mean a diagram from https://github.com/agiorguk/gemini/blob/main/resources/GEMINI%20UML.eapx - but I no longer have  an EA license to open it. Could someone export a relevant diagram in a format good for embedding in perhaps https://github.com/agiorguk/gemini/blob/main/docs/1037-uk-gemini-standard-and-inspire-implementing-rules.asciidoc

**Comment by archaeogeek** [comment]
James to see if he has an appropriate license

**Comment by nmtoken** [comment]
I've converted the eapx file and can access, but not sure which of the many UML figures gives what is needed.

View of comprehensive dataset metadata profile (19115 Edition 1)

![Image](https://github.com/user-attachments/assets/4c2fa223-cb75-4f69-97fa-f43446a4054d)

Metadata all?

Just wondering too whether we are allowed to share the ISO UML diagrams?

**Comment by archaeogeek** [comment]
Unclear whether this diagram actually helps with the original question of inheritance/scoping of metadata to all elements. Also, more generally, is it clearer for a non-technical person to understand than our existing guidance?

**Comment by PeterParslow** [comment]
I agree it may not help #24, that's just where the idea came up. I think it's more of a help to technical people starting to grapple with GEMINI!
